msg/en_US/utils.msg

```diff
@@ -978,9 +978,9 @@ usage: %1$s applyinfo [OPTION] database-name\n\
 \n\
 valid options:\n\
   -r, --remote-host-name      remote host name; display remote node's active log information \n\
-  -a, --applied-info 	      display applied information \n\
+  -a, --applied-info 	      display applied information; the -L option is required \n\
   -L, --copied-log-path=PATH  path of copied log volumes; display copied log information \n\
-  -p, --page=ID               page id; default : 0\n\
+  -p, --page=ID               page id; default : 0; the -L option is required \n\
```

Comment:
```suggestion
  -p, --page=ID               page id; the -L option is required; default : 0 \n\
```

It seems better to show the `default` at the end of the message.

Reply:
oh, @wrlawodms left a same review. if you change this line, please just resolve this conversation.